libyul/AssemblyStack.h

```diff
@@ -124,9 +124,9 @@ class AssemblyStack: public langutil::CharStreamProvider
 private:
 	bool analyzeParsed();
 	bool analyzeParsed(yul::Object& _object);
-
+public:
```

Comment:
Is this needed or not? If yes, where is it needed?

Reply:
Indeed also works via object compiler - fixed.